### PR TITLE
Solution of Leetcode Problem: 98 

### PR DESCRIPTION
**Pull request by Anshika14528**
## Related Issue

closes: #872

## Description

Added solution to leetcode problem 98 (Validate Binary Search Tree)

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/0000-0099/0098-validate-binary-search-tree.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+```
+Input: root = [5,1,4,null,null,3,6]
+Output: false
+```
+
+### Constraints
+
+- The number of nodes in the tree is in the range $[1, 10^4]$.
+- -2^(31) <= Node.val <= 2^(31) - 1.
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0098-validate-binary-search-tree.md:43]: replace `- -2^(31) <= Node.val <= 2^(31) - 1.` to `- $-2^(31) <= Node.val <= 2^(31) - 1$`.

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by Anshika14528** [comment]
Done
please review it now @Ajay-Dhangar sir

**Comment by Anshika14528** [comment]
@Ajay-Dhangar sir , i have corrected the changes requested please review it now

File: dsa-solutions/lc-solutions/0000-0099/0098-validate-binary-search-tree.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+id: validate binary search tree
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0098-validate-binary-search-tree.md:2]: replace `validate binary search tree` to `validate-binary-search-tree`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update based on suggestions

**Comment by Anshika14528** [comment]
@Ajay-Dhangar sir, is it ok now i have made the chnages

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Anshika14528** [comment]
@Ajay-Dhangar sir , why is it duplicate ?

**Comment by Anshika14528** [comment]
My all three issues solution(issue no. #872 , #883) was there in this pull request "Added leetcode problem:226 #913"

**Pull request merged**